package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "tscript",
+  "version": "1.0.0",
+  "description": "",
+  "main": "tscript.js",
+  "scripts": {
+    "build": "webpack --mode=production",
+    "dev": "webpack --mode=development -w -c ./webpack.config.js",
```

```suggestion
    "dev": "webpack --mode=development --watch",
```

* `--watch` can be easier recognized than `-w`
* specifying the config file is not necessary